40.meta-mender/docs.md

```diff
@@ -7,6 +7,98 @@ shortcode-core:
 github: false
 ---
 
+## meta-mender zeus-v2021.07
+
+_Released 07.23.2021_
+
+### Statistics
+
+A total of 1314 lines added, 298 removed (delta 1016)
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 12 (40.0%) |
+| Lluis Campos | 11 (36.7%) |
+| Ole Petter Orhagen | 2 (6.7%) |
+| Fabio Tranchitella | 2 (6.7%) |
+| Pierre-Jean Texier | 1 (3.3%) |
+| Corey Cothrum | 1 (3.3%) |
+| Alan Martinovic | 1 (3.3%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Kristian Amlie | 1009 (76.6%) |
+| Lluis Campos | 238 (18.1%) |
+| Fabio Tranchitella | 55 (4.2%) |
+| Ole Petter Orhagen | 12 (0.9%) |
+| Pierre-Jean Texier | 1 (0.1%) |
+| Corey Cothrum | 1 (0.1%) |
+| Alan Martinovic | 1 (0.1%) |
+
+| Developers with the most signoffs (total 6) | |
+|---|---|
+| Lluis Campos | 6 (100.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 28 (93.3%) |
+| texier.pj2@gmail.com | 1 (3.3%) |
+| contact@coreycothrum.com | 1 (3.3%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 1315 (99.8%) |
+| texier.pj2@gmail.com | 1 (0.1%) |
+| contact@coreycothrum.com | 1 (0.1%) |
+
+| Employers with the most signoffs (total 6) | |
+|---|---|
+| Northern.tech | 6 (100.0%) |
+
+| Employers with the most hackers (total 7) | |
+|---|---|
+| Northern.tech | 5 (71.4%) |
+| texier.pj2@gmail.com | 1 (14.3%) |
+| contact@coreycothrum.com | 1 (14.3%) |
+
+### Changelogs
+
+#### meta-mender (zeus-v2021.07)
+
+New changes in meta-mender since zeus-v2021.04:
+
+* Add meta-oe as dependency to mender-demo
+* mender-client-dev package to distribute D-Bus API specs
+  ([MEN-4106](https://tracker.mender.io/browse/MEN-4106))
+* fix grub chownboot task ordering
+* Add io.mender.UpdateManager to D-Bus policy files.
+* Fix build error when trying to build mender-configure
+  without systemd:
+  ```
+  mender-configure-1.0.0-r0 do_package: SYSTEMD_SERVICE_mender-configure value mender-configure-apply-device-config.service does not exist.
+  ```
+* u-boot-mender: Ignore case when checking variables in mender_Kconfig_fragment
```

Comment:
suggested change "Ignore case" -> "Ignore _the_ case" (or is it taken from the commit messages?)

Reply:
I cannot rephrase a commit message :) Only in very serious typos I would edit the auto-generated text